tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -205,23 +206,30 @@ Object.Dai.SSP [
 # PCM2 ---> gain ---> SRC ---> Mixin  ---> Mixout ---> gain ---> SSP2
 # PCM4 ---> gain ---> Mixin -------------------------->\
 # PCM3 ---> gain ---> Mixin ---->\                      \
-# PCM0 ---> gain ---> Mixin ---> Mixout ---> Mixin ---> Mixout ---> gain ---> smart_amp ---> SSP0
-#                                                                             /
-#    /---------------------------------------------------------------------->/
-# SSP0 ---> PCM0
+# PCM0 ---> micsel ---> gain ---> Mixin ---> Mixout ---> Mixin ---> Mixout ---> gain ---> smart_amp ---> SSP0
+#                                                                                         /
+#    /---------------------------------------------------------------------------------->/
+# SSP0 ---> gain ---> module-copier ---> gain ---> micsel ---> PCM0
+#                                 \----> gain ---------------> PCM0.1
```

Comment:
Remove the ASCII diagram update for capture, there's no capture micsel in this version.